src/main/java/racingcar/view/ResultView.java

```diff
@@ -1,35 +1,48 @@
 package racingcar.view;
 
-import racingcar.model.RaceRecord;
+import racingcar.model.Car;
 import racingcar.model.RaceWinner;
 
 import java.util.List;
+import java.util.stream.Collectors;
 
 public class ResultView {
 
     public ResultView() {
         System.out.println("실행 결과");
     }
 
-    public void printRacingResult(List<RaceRecord> records, int raceCount, int totalNumberOfRace) {
-
-        for (RaceRecord record : records) {
-            System.out.println(record.getCarName() + " : " + generateDashedString(record.getCarMovingDistance()));
+    public void printRacingResult(List<Car> cars, int raceCount, int totalNumberOfRace) {
+        for (Car car : cars) {
+            System.out.println(car.getName() + " : " + generateDashedString(car.getMovingDistance()));
         }
 
         System.out.println();
 
         if (raceCount == totalNumberOfRace) {
-            printRaceWinner(records);
+            printRaceWinner(cars);
         }
     }
 
     private String generateDashedString(int movingDistance) {
         return "-".repeat(movingDistance);
     }
 
-    private void printRaceWinner(List<RaceRecord> records) {
-        RaceWinner winner = new RaceWinner(records);
-        System.out.println(winner.getNames());
+    private void printRaceWinner(List<Car> cars) {
+        RaceWinner winner = new RaceWinner(cars);
+        String winnerNames = winnerListToString(winner.getWinnerList());
+
+        System.out.println(winnerNames);
+    }
+
+    private String winnerListToString(List<Car> winnerList) {
```

Comment:
```suggestion
    private String winnersToString(List<Car> winners) {
```

리스트 대신 복수형을 사용하신다면 더 좋을 것 같습니다.
자료구조가 크게 변할 가능성은 적더라도, 네이밍에서 자료구조에 의존하는 것은 좋은 선택이 아니라고 생각해요!